RunTest.bat

```diff
@@ -135,9 +135,9 @@ if "%all%" == "yes" (
   set do7=yes
   set do8=yes
   set do9=yes
-  set do10=yes
+  set do10=no
   set do11=yes
-  set do12=yes
+  set do12=no

```

Comment:
This is my worry. If something does not work, we might not be able to do anything with it, just disable tests. CI is usually good for actively developed targets.

Reply:
well, we don't have yet functionality that allow us to debug remotely broken tests, which is why the only option now is to disable them.

but I happened to already build that for windows just because I was curious on why those 2 tests were failing and you can see both the code and a run with a downloadable set of files showing the output in:

  https://github.com/carenas/pcre2/actions/runs/2199761640

there is also a tentative solution for Linux there, and of course fixes pending for the real underlying issue (which I am still puzzled about, because it will seem to be an extra empty line from pcre2test, and is consistently broken in that old OS github provides for its action VM)